supervisor/dbus/network/interface.py

```diff
@@ -49,7 +49,7 @@ def object_path(self) -> str:
 
     @property
     @dbus_property
-    def name(self) -> str:
+    def interface_name(self) -> str:
```

Comment:
`name` is also defined in the base DBusProxy object all these classes used so this override is not allowed. I guess prior versions of mypy didn't catch it. This is probably the only change in here that worries me a bit but I carefully combed through references and search results and am pretty sure I got all of them.